src/meta/ee/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = { workspace = true }
 
 [lib]
 doctest = false
-test = false
+test = true
```

Comment:
Since some of our crates disabled lib tests with `test = false`, I'd like to be more precise and explicit about that there are lib tests in this crate.

Reply:
Makes sense.